src/services/index.js

```diff
@@ -5,3 +5,4 @@ module.exports.contactService = require('./contact.service');
 module.exports.productService = require('./product.service');
 module.exports.categoryService = require('./category.service');
 module.exports.messageService = require('./message.service');
+module.exports.cartDetailService = require('./CartDetail.service');
```

Comment:
- sửa hết viết thường chữ cái đầu
-
```suggestion
module.exports.cartDetailService = require('./cartDetail.service');
```